modules/toast/react/lib/Toast.tsx

```diff
@@ -97,13 +97,20 @@ export default class Toast extends React.Component<ToastProps> {
         padding={PopupPadding.s}
         handleClose={onClose}
         closeIconSize={IconButtonSize.Small}
+        role={!onClose && !onActionClick ? 'status' : 'dialog'}
+        aria-live={!onClose && !onActionClick ? 'polite' : undefined}
```

Comment:
Similar to `aria-atomic`, I wonder if this should be `'polite' : 'off'`. It's very possible I'm missing something or I'm incorrect. 😄

Reply:
I guess I didn't know what the opposite of polite was to be fair